cfgov/unprocessed/apps/admin/js/rich-text-table.js

```diff
@@ -88,7 +88,7 @@ function richTextTable(Handsontable) {
 
     /* TextEditor's TEXTAREA will hold our data for Draftail, and we'll use
        its style when we open the editor. */
-    this.TEXTAREA.id = 'rich-text-table-cell-editor';
+    this.TEXTAREA.id = this.TEXTAREA_PARENT.parentElement.id + '-rich-text-table-cell-editor';
```

Comment:
This looks fine if the grandparent ID is always unique. If you encounter issues and need a guaranteed GUID there is also https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

Reply:
I actually tried that, but it seems that UUIDs aren't (always?) valid selectors?

The grandparent id includes the id that gets passed to our `initAtomicTable` function, so it looks like `content-4-value-handsontable-container` for example, which means the Draftail textarea id ends up with `content-4-value-handsontable-container-rich-text-table-cell-editor`.